CMakeLists.txt

```diff
@@ -14,6 +14,14 @@ endif()
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/libraries/fc/CMakeModules")
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/CMakeModules")
 
+if (UNIX)
+   if (APPLE)
+      if (LLVM_DIR STREQUAL "" OR NOT LLVM_DIR)
+         set(LLVM_DIR "/usr/local/Cellar/llvm@4/4.0.1/lib/cmake/llvm")
```

Comment:
Where is LLVM_DIR used? I can't seem to find any other reference to it

Reply:
It is used in FindLLVM.cmake.

Reply:
okay yes I see, LLVM_DIR is used as an input hint to find_package(LLVM)